app/javascript/components/admin/ActiveUsers.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import useActiveUsers from '../../hooks/queries/admins/useActiveUsers';
+import ManageUsersTable from './ManageUsersTable';
+
+export default function ActiveUsers() {
+  const { data: users } = useActiveUsers();
+
+  return (
+    <ManageUsersTable users={users} />
+  );
+}
```

Comment:
I made it this way looking into the future.

The reason I did this is so that when we implement the other tabs such as 'Pending', we can re-use the <ManageUsersTable> and just pass in the pending 'Users' into it. It cleans up the code and gets rid of repetition

Reply:
Right. Nicely done!